Lib/test/test_pprint.py

```diff
@@ -453,13 +453,22 @@ class AdvancedNamespace(types.SimpleNamespace): pass
                   dog=8)""")
 
     def test_subclassing(self):
+        # length(repr(obj)) > width
         o = {'names with spaces': 'should be presented using repr()',
              'others.should.not.be': 'like.this'}
         exp = """\
 {'names with spaces': 'should be presented using repr()',
  others.should.not.be: like.this}"""
         self.assertEqual(DottedPrettyPrinter().pformat(o), exp)
 
+        # length(repr(obj)) < width
+        o1 = ['with space']
+        exp1 = "['with space']"
+        self.assertEqual(DottedPrettyPrinter().pformat(o1), exp1)
+        o2 = ['without.space']
+        exp2 = "[without.space]"
+        self.assertEqual(DottedPrettyPrinter().pformat(o2), exp2)
```

Comment:
I don't think `DottedPrettyPrinter()` should be called three times in this test method.